controllers/pages.controller.js

```diff
@@ -0,0 +1,82 @@
+const nodemailer = require("nodemailer");
+
```

Comment:
why do you have a period in between the pages and the controller and the name pages, please agree is quite generic, please change it to something a bit more self explanatory like informationController of AboutUsController
@eladaviv